packages/Webkul/Checkout/src/Cart.php

```diff
@@ -137,6 +135,9 @@ public function addProduct($productId, $data)
         }
 
         $product = $this->productRepository->findOneByField('id', $productId);
+        if ($product->status === 0) {
```

Comment:
blank line should be added before if statement.

Reply:
Done